components/contribution-flow/StepDetails.js

```diff
@@ -124,6 +124,14 @@ const StepDetails = ({ onChange, stepDetails, collective, tier, showPlatformTip,
 
       {!isFixedContribution ? (
         <Box mb="30px">
+          {amount && amount < minAmount && (
+            <div className="mb-2 text-red-400">
+              <FormattedMessage
+                id="amount.belowMinimum"
+                defaultMessage={`Please enter an amount of ${formatCurrency(minAmount, currency)} or more`}
```

Comment:
Formatting must be done through props for internationalization to work, see https://formatjs.github.io/docs/react-intl/components#formattedmessage